tests/testthat/test-theme.R

```diff
@@ -293,7 +293,7 @@ test_that("incorrect theme specifications throw meaningful errors", {
 test_that("element tree can be modified", {
   # we cannot add a new theme element without modifying the element tree
   p <- ggplot() + theme(blablabla = element_text(colour = "red"))
-  expect_snapshot_error(print(p))
```

Comment:
Wouldn't this require an update of the snapshot

Reply:
The message is exactly the same so the snapshot file hasn't changed

Reply:
👍